integration_tests/src/main/python/spark_init_internal.py

```diff
@@ -12,18 +12,44 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from pyspark.sql import SparkSession
+try:
+    from pyspark.sql import SparkSession
+except ImportError as error:
+    import findspark
+    findspark.init()
+    from pyspark.sql import SparkSession
 
 def _spark__init():
     #Force the RapidsPlugin to be enabled, so it blows up if the classpath is not set properly
     # DO NOT SET ANY OTHER CONFIGS HERE!!!
     # due to bugs in pyspark/pytest it looks like any configs set here
     # can be reset in the middle of a test if specific operations are done (some types of cast etc)
     # enableHiveSupport() is needed for parquet bucket tests
-    _s = SparkSession.builder \
+    #TODO need to figure out a better way to do this optionally
+    import os
+    _sb = SparkSession.builder
+    _sb.config('spark.master', 'local') \
```

Comment:
yeah definitely an issue that can't point to any cluster, we definitely want that especially with emr and dataproc testing.  we could also just have this entire thing configured on and off easily, with it off by default so it would just work if people don't configure it properly.